includes/class-order.php

```diff
@@ -200,13 +248,70 @@ public static function order_status( $order_id ) {
 
 
 	/**
-	 * Process order status response.
+	 * Get order extra data.
 	 *
-	 * @param  int                  $order_id Order ID.
-	 * @param  \Zotapay\ApiResponse $response Response Status.
+	 * @param  \Zotapay\ApiCallback $callback API Response.
+	 * @return array|false
+	 */
+	public static function get_extra_data( $callback ) {
+		// Check extra data.
+		if ( empty( $callback->getExtraData() ) ) {
+			return false;
+		}
+
+		return $callback->getExtraData();
+	}
+
+
+	/**
+	 * Is order amount changed.
+	 *
+	 * @param  \Zotapay\ApiCallback $callback API Response.
+	 * @return array|false
+	 */
+	public static function amount_changed( $callback ) {
+		$extra_data = self::get_extra_data( $callback );
+
+		// If no extra data return.
+		if ( empty( $extra_data ) ) {
+			return false;
+		}
+
+		// Check if has amount changed key.
+		if ( false === isset( $extra_data['amountChanged'] ) ) {
```

Comment:
Maybe just use `empty( $extra_data['amountChanged'] )` here? This will free us from those Yoda conditions :-)

Reply:
I wanted to be more precise but removing Yoda is good idea.